packages/stark-ui/src/util/component/component.util.spec.ts

```diff
@@ -26,4 +28,70 @@ describe("Util: ComponentUtil", () => {
 			expect(result).toBe(false);
 		});
 	});
+
+	describe("isOutputWiredUp", () => {
+		let component: TestComponent;
+
+		beforeEach(async(() => {
+			return TestBed.configureTestingModule({
+				declarations: [TestHostWiredUpComponent, TestHostNotWiredUpComponent, TestComponent]
+			}).compileComponents();
+		}));
+
+		it("shouldReturnTrue", () => {
+			const hostFixture: ComponentFixture<TestHostWiredUpComponent> = TestBed.createComponent(TestHostWiredUpComponent);
+			const hostComponent: TestHostWiredUpComponent = hostFixture.componentInstance;
+			hostFixture.detectChanges(); // trigger initial data binding
+			component = hostComponent.testComponent;
+
+			expect(StarkComponentUtil.isOutputWiredUp(component.eventEmitter)).toBe(true);
+		});
+
+		it("shouldReturnFalse", () => {
```

Comment:
Maybe a more descriptive test will be useful... when it is supposed to return false? ;)